sdk/cmake/msvc.cmake

```diff
@@ -51,7 +51,12 @@ endif()
 # HACK: for VS 11+ we need to explicitly disable SSE, which is off by
 # default for older compilers. See CORE-6507
 if(ARCH STREQUAL "i386")
-    add_compile_flags("/arch:IA32")
+    # Clang's IA32 means i386, which doesn't have cmpxchg8b
```

Comment:
So what kind of value Clang is actually using (what's the value of ${OARCH}) when RosBE's specified ARCH is i386?

Reply:
> which doesn't have cmpxchg8b

But this instruction is required for ReactOS to work, isn't it?

Reply:
It defaults to `pentium` but can be overridden. https://github.com/reactos/reactos/blob/master/sdk/cmake/config.cmake#L6

Yes, `cmpxchg8b` is required, which is exactly why things won't work with `i386` and require `pentium`.